content/source/docs/enterprise/sentinel/index.html.md

```diff
@@ -22,6 +22,10 @@ policy using Terraform itself.
 
 ## Sentinel in Terraform Enterprise
 
+~> Sentinel now supports native VCS integration and direct policy set uploads.
+   See the [manage policies](./manage-policies.html) page for details or to read
+   about the [migration utility](./manage-policies.html#migration-utility).
```

Comment:
I'd refer to the page that we also lists below with the same title to avoid confusion; maybe:

"See [Managing policies for organizations]() below for details, including a [migration utility]()."